modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -838,33 +838,38 @@ private CompletableFuture<Void> close(boolean cancel) {
                                 : closeLocalFragments();
 
                         var finalStepFut = cancelResult.whenComplete((r, e) -> {
+                            if (e != null) {
+                                Throwable ex = unwrapCause(e);
+
+                                LOG.warn("Fragment closing processed with errors, root [queryId={}]", ex, ctx.queryId());
```

Comment:
not sure why do we need "root" at the end

Reply:
it`s all about "self review" )